### DISCUSSION TITLE
Configure system-bridge

### DISCUSSION THREAD

**Discussion by florian-asche** [timmo001/system-bridge#3099] [Ideas]
Is it possible to configure system-bridge? Can i disable specific modules? I want to use it to get some basic data and shutdown my computers. I dont want to share files or publish unwanted data.

**Comment by timmo001** [comment]
Not a bad idea, it would require some refactoring to disable certain modules/features

**Comment by akentner-cp** [comment]
Another idea for configuring: allow to exclude devices like `/snap/*` or several `/run/*`. I don't want to see this in Home Assistant

**Reply by timmo001** [reply]
You can always disable the entities you dont want in Home Assistant
